types/matter-js/index.d.ts

```diff
@@ -3691,6 +3739,7 @@ declare namespace Matter {
          * @event afterRender
          * @param {} event An event object
          * @param {number} event.timestamp The engine.timing.timestamp of the event
+         * @param {number} event.delta The delta time in milliseconds value used in the update
```

Comment:
Incorrect, property is not available, see https://github.com/liabru/matter-js/blob/0.20.0/src/render/Render.js#L1564-L1568